src/MongoDB.Bson/MongoDB.Bson.csproj

```diff
@@ -56,4 +56,8 @@
     <Compile Include="..\MongoDB.Shared\Hasher.cs" Link="Hasher.cs" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Include="..\..\License.txt" Pack="true" PackagePath="$(PackageLicenseFile)" />
+  </ItemGroup>
+
```

Comment:
This snippet includes the License.txt file in the project so that PackageLicenseFile can reference it.